tests/e2e/local/minikube/README.md

```diff
@@ -57,6 +58,26 @@ To cleanup host settings only (remove docker daemon setup and port forwarding)
 ```bash
 . ./cleanup_host.sh
 ```
+### Debug with KubeSquash
```

Comment:
Sorry same question, does this apply to vagrant too?

Reply:
It should but have not tested with vagrant.. will test and update vagrant?

Reply:
yeah if it works with vagrant too, we can move it up one level, similar to the tips doc..